### PR TITLE
docs: reshape wording

### PR DESCRIPTION
**Pull request by GangGreenTemperTatum**
thank you for the awesome contribution!

just a small nitpick if anything, i had to recheck the code a few times and reverse engineer the logic to confirm my theory as the default "Evaluated X PRs" was super throwing me off here 🙂 
suggestive change to add some ambiguity to prevent this potentially happening in the future, hope that you are happy with the wording etc! i have another PR coming after this for a working Slack integration but i thought i'd pipe this in first to prevent merge conflicts

### CODE REVIEW

**Comment by GangGreenTemperTatum** [comment]
hey @henry-prince-addepar @addepar-tg , any updates on this? sorry to chase, been a while :) tyia!

File: README.md [hunk 1]
```diff
@@ -16,7 +16,7 @@
 
 RedFlag leverages AI to determine high-risk code changes. 
 Run it in batch mode to scope manual security testing of release candidates, 
-or run it in your CI pipelines to flag PRs and add the appropriate reviewers. 
+or run it in your CI pipelines to flag commits, either spanning a single or multiple PRs and add the appropriate reviewers. 
 Despite being a security tool, RedFlag can be leveraged for almost any team
```
Comment (addepar-tg) [inline README.md]: It's really made to flag PRs in this mode - by using the Action, RedFlag is fed a PR number and it evaluates the PR title and files changed in the PR.

File: README.md [hunk 2]
```diff
@@ -137,8 +145,8 @@ By default, RedFlag produces an HTML report that can be opened in a browser.
 
 # CI Mode
 
-RedFlag can be run in CI pipelines to flag PRs and add the appropriate reviewers.
```
Comment (addepar-tg) [inline README.md]: Again this is more accurate as in this mode, RedFlag evaluates details of the PR (nothing more, nothing less).

Comment (GangGreenTemperTatum) [reply README.md]: good feedback, I saw you committed so thanks! :)

File: README.md (hunk 1 above)
Comment (GangGreenTemperTatum) [reply README.md]: good feedback, I saw you committed so thanks! :)

**Comment by addepar-tg** [comment]
Thanks for the contribution @GangGreenTemperTatum! Will get another pair of eyes (@henry-prince-addepar) on the combined changes before committing.

**Comment by GangGreenTemperTatum** [comment]
> Thanks for the contribution @GangGreenTemperTatum! Will get another pair of eyes (@henry-prince-addepar) on the combined changes before committing.

Thanks guys! Hopefully you like the other 2 feature PRs I submitted too :)

**Review by henry-prince-addepar — APPROVED** [review]
LGTM

Thanks, @GangGreenTemperTatum and @addepar-tg.

**Comment by GangGreenTemperTatum** [comment]
dope! ty @addepar-tg @henry-prince-addepar , hope you enjoy the other PRs and lmk if you need anything :)

**Pull request merged**